examples/failing/QualifiedQualifiedImports.purs

```diff
@@ -0,0 +1,9 @@
+-- @shouldFailWith ErrorParsingModule
+module Main where
+
+import Prelude
+
+-- qualified import with qualified imported names
+import qualified Control.Monad.Eff.Console (log) as Console
```

Comment:
It was an untested feature, so I figured it was accidentally allowing this form since I didn't consider this use case. I'll move this to the passing tests and make it work again.